### PR TITLE
Create GearCardConnection component

### LINKED ISSUE #15: Create gear card connection section
- Add h3
- Add random cable svg
- Add styling

### PR DESCRIPTION
**Pull request by Seborider**
GearCardConnection component is later needed as a building brick for the GearCard component.

- write a random function to randomly spawn svg components
- add styling

Resolves #15

### CODE REVIEW

File: src/app/components/GearCardConnection/GearCardConnection.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import Cable_2 from '../assets/cables/Cable2_Icon';
+import Cable_3 from '../assets/cables/Cable3_Icon';
+import Cable_4 from '../assets/cables/Cable4_Icon';
+import Cable_5 from '../assets/cables/Cable5_Icon';
+import Cable_6 from '../assets/cables/Cable6_Icon';
+import Cable_7 from '../assets/cables/Cable7_Icon';
+import Typography from '../Typography/Typography';
+import style from './GearCardConnection.module.css';
+
+type GearConnectionProps = {
```
Comment (marwinburesch) [inline src/app/components/GearCardConnection/GearCardConnection.tsx:12]: This doesn't match the filename or the component name

File: src/app/components/GearCardConnection/GearCardConnection.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  <Cable_1 fill="var(--navigation-icon-color)" />,
+  <Cable_2 fill="var(--navigation-icon-color)" />,
+  <Cable_3 fill="var(--navigation-icon-color)" />,
+  <Cable_4 fill="var(--navigation-icon-color)" />,
+  <Cable_5 fill="var(--navigation-icon-color)" />,
+  <Cable_6 fill="var(--navigation-icon-color)" />,
+  <Cable_7 fill="var(--navigation-icon-color)" />,
+];
+
+export default function Cable1({
```
Comment (marwinburesch) [inline src/app/components/GearCardConnection/GearCardConnection.tsx:33]: This also seems to be incorrectly named :)

**Review by marwinburesch — COMMENTED** [review]
💯 !

File: src/app/components/GearCardConnection/GearCardConnection.tsx (hunk 1 above)
Comment (Seborider) [reply src/app/components/GearCardConnection/GearCardConnection.tsx]: Resolved in [76b9e9b](https://github.com/Seborider/plugs./pull/33/commits/76b9e9b7203ba3524422a0f2dfbae26dfe148aa8)

File: src/app/components/GearCardConnection/GearCardConnection.tsx (hunk 2 above)
Comment (Seborider) [reply src/app/components/GearCardConnection/GearCardConnection.tsx]: Resolved in [76b9e9b](https://github.com/Seborider/plugs./pull/33/commits/76b9e9b7203ba3524422a0f2dfbae26dfe148aa8)

**Pull request merged**